components/base/sensorcontrolled/sensorcontrolled.go

```diff
@@ -255,6 +262,20 @@ func (sb *sensorBase) Geometries(ctx context.Context, extra map[string]interface
 	return sb.controlledBase.Geometries(ctx, extra)
 }
 
+func (sb *sensorBase) DoCommand(ctx context.Context, req map[string]interface{}) (map[string]interface{}, error) {
+	resp := make(map[string]interface{})
+
+	sb.mu.Lock()
+	defer sb.mu.Unlock()
+	ok := req[getPID].(bool)
+	if ok {
+		resp[getPID] = fmt.Sprintf(`tuned PID vals: {p: %v, i: %v, d: %v, type: linear_velocity}, {p: %v, i: %v, d: %v, type: angular_velocity}`,
+			(*sb.tunedVals)[0].P, (*sb.tunedVals)[0].I, (*sb.tunedVals)[0].D, (*sb.tunedVals)[1].P, (*sb.tunedVals)[1].I, (*sb.tunedVals)[1].D)
+	}
```

Comment:
could maybe use the function for the DoCommand too, but that might be overkill

Reply:
going to leave them separate as the formatting is a bit different with an interface vs an error/string